packages/web-provider/src/universal-provider/types.ts

```diff
@@ -0,0 +1,86 @@
+import { EventEmitter } from 'events'
```

Comment:
I'm not so sure we should add EventEmitter. It's nodejs native and we may get troubles in browser context. But if it works fine now, let it be.